php/values.yaml

```diff
@@ -293,22 +295,12 @@ fpm:
   lifecycle:
     postStart: []
     # postStart: ["/bin/sh", "-c", "echo Hello > /tmp/message"]
+    # sleep 0 <= processControlTimeout
+    preStop: ["/bin/sh", "-c", "sleep 1; kill -QUIT 1; sleep 0"]
```

Comment:
@cw-niinuma 
I think that an empty array is better.
The reason is that `processControlTimeout` is 0, so there is no need to perform a safe shutdown.
https://github.com/chatwork/charts/pull/109/files#diff-18bf1c8a8cbc10309f2eca7c8c03fde3R518